Update to test with go-ipfs 0.7.0

Released September 22, 2020: https://github.com/ipfs/go-ipfs/releases/tag/v0.7.0

I mistaken, as is evident in https://travis-ci.org/github/oduwsdl/ipwb/jobs/729743815.

The version check is for < 0.7.0, not ≤ 0.7.0

```
E     ipfshttpclient.exceptions.VersionMismatch: Unsupported daemon version '0.7.0' (not in range: 0.4.22 ≤ … < 0.7.0)
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -17,10 +17,13 @@ jobs:
         python:
           - 3.7
           - 3.8
+          - 3.9
         ipfs:
           - 0.4
           - 0.5
           - 0.6
+          - 0.7
+          - 0.8
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.os }} Py-${{ matrix.python }} IPFS-${{ matrix.ipfs }}
     steps:

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ python:
   - "3.7"
   - "3.8"
 before_script:
-  - wget "https://dist.ipfs.io/go-ipfs/v0.6.0/go-ipfs_v0.6.0_linux-amd64.tar.gz" -O /tmp/ipfs.tar.gz
+  - wget "https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz" -O /tmp/ipfs.tar.gz
   #- mkdir $HOME/bin
   - pushd . && cd $HOME/bin && tar -xzvf /tmp/ipfs.tar.gz && popd
   - export PATH="$HOME/bin/go-ipfs:$PATH"

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 warcio>=1.5.3
-ipfshttpclient>=0.6.1
+ipfshttpclient>=0.8.0a
 Flask==1.1.1
 pycryptodome>=3.4.11
 requests>=2.19.1

